src/g_game.c

```diff
@@ -100,6 +101,7 @@ int             starttime;     // for comparative timing purposes
 boolean         viewactive;
 int             deathmatch;    // only if started as net death
 boolean         netgame;       // only true if packets are broadcast
+boolean         solonet;
```

Comment:
I think we can remove the `solonet` checks, no timer adjustments goin' on here, so it works like single player.

Reply:
Ah, I get it. Thank you.